ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/ipv6-neighbors.j2

```diff
@@ -0,0 +1,27 @@
+{#
+ Copyright (c) 2023-2024 Arista Networks, Inc.
+ Use of this source code is governed by the Apache License 2.0
+ that can be found in the LICENSE file.
+#}
+{# eos - ipv6 neighbors #}
+{% if ipv6_neighbor.cache is arista.avd.defined or ipv6_neighbor.persistent is arista.avd.defined or ipv6_neighbor.static_entries is arista.avd.defined %}
+!
+{%     if ipv6_neighbor.cache is arista.avd.defined(true) %}
+ipv6 neighbor cache persistent
+{%     endif %}
+{%     if ipv6_neighbor.persistent is arista.avd.defined(true) %}
+ipv6 neighbor persistent refresh-delay {{ ipv6_neighbor.refresh_delay | arista.avd.default('600') }}
+{%     endif %}
```

Comment:
Something seems off here. cache persistent vs persistent refresh-delay. The data model does not correspond to the CLI. Please revisit.

Reply:
I think it is correct, the CLI is not really good here. There are two commands where the help tells the same:

```
B1-LF-1010(config)#ipv6 neighbor ?
 ...
  persistent       Restore the IPv6 neighbor cache after reboot
```

```
B1-LF-1010(config)#ipv6 neighbor cache ?
  persistent  Restore the IPv6 neighbor cache after reboot
```

But when you actually configure `ipv6 neighbor persistent` it will put only the following into the config:
`ipv6 neighbor persistent refresh-delay 600`

so in my perspective the data model is correct and the EOS CLI is confusing.

Reply:
So a bit of renaming in the model will bring it closer to the CLI and more in line with what we do elsewhere:
```
cache -> cache_persistent
refresh_delay -> persistent_refresh_delay
```

Reply:
Actually we should *not* have a default value for the refresh delay, and only add it in the config if given by the user. We have a general rule of not implementing EOS defaults in our code. If the user wants it printed, they should set the value.

Reply:
updated the model